packages/react-components/react-carousel/library/src/components/CarouselContext.types.ts

```diff
@@ -17,6 +19,7 @@ export type CarouselIndexChangeData = (
 
 export type CarouselContextValue = {
   activeIndex: number;
+  appearance: CarouselAppearance;
```

Comment:
This cannot be required, will be a BC on typings

```suggestion
  appearance?: CarouselAppearance;
```